server.js

```diff
@@ -5,6 +5,11 @@ if (typeof userFunctionUri === "undefined" || userFunctionUri.trim() === "") {
     throw "FUNCTION_URI envvar not set or empty. Aborting.";
 }
 
+const debug = process.env.NODE_DEBUG;
+if (typeof debug === "undefined" || debug !== "riff") {
```

Comment:
`NODE_DEBUG` is a comma-separated set of debug loggers, we can't assume exclusive use